pkgs/development/tools/misc/pkgconf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, automake, autoconf, libtool }:
+
+stdenv.mkDerivation rec {
+  name = "pkgconf-1.5.4";
+
+  src = fetchgit {
+    url = "https://git.dereferenced.org/pkgconf/pkgconf.git";
```

Comment:
No need to quote simple URLs.

Reply:
The website points to https://distfiles.dereferenced.org/pkgconf/pkgconf-1.1.0.tar.xz

Reply:
See the bottom of the readme.